docs/docs/how-tos/sequence.ipynb

```diff
@@ -0,0 +1,327 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How to create a sequence of steps\n",
+    "\n",
+    "!!! info \"Prerequisites\"\n",
+    "    This guide assumes familiarity with the following:\n",
+    "\n",
+    "    - [How to define and update graph state](../../how-tos/state-reducers)\n",
+    "\n",
+    "This guide demonstrates how to construct a simple sequence of steps. We will demonstrate:\n",
+    "\n",
+    "1. How to build the graph\n",
+    "2. Built-in short-hand for constructing similar graphs.\n",
+    "\n",
+    "\n",
+    "<details>\n",
+    "<summary>Why split application steps into a sequence with LangGraph?</summary>\n",
+    "\n",
+    "The primary benefit of using LangGraph for this case is access to the underlying persistence layer. LangGraph\n",
+    "checkpoints state in between the execution of nodes, so your LangGraph nodes govern:\n",
+    "\n",
+    "<ul>\n",
+    "<li>How state updates are [checkpointed](../../concepts/persistence/)</li>\n",
+    "<li>How interruptions are resumed in [human-in-the-loop](../../concepts/human_in_the_loop/) workflows</li>\n",
+    "<li>How we can \"rewind\" and branch-off executions using LangGraph's [time travel](../../concepts/time-travel/) features</li>\n",
+    "</ul>\n",
+    "\n",
+    "They also determine how execution steps are [streamed](../../concepts/streaming/), and how your application is visualized\n",
+    "and debugged using [LangGraph Studio](../../concepts/langgraph_studio/).\n",
+    "\n",
+    "</details>\n",
+    "\n",
+    "## Setup\n",
+    "\n",
+    "First, let's install langgraph:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%capture --no-stderr\n",
+    "%pip install -U langgraph"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"admonition tip\">\n",
+    "     <p class=\"admonition-title\">Set up <a href=\"https://smith.langchain.com\">LangSmith</a> for better debugging</p>\n",
+    "     <p style=\"padding-top: 5px;\">\n",
+    "         Sign up for LangSmith to quickly spot issues and improve the performance of your LangGraph projects. LangSmith lets you use trace data to debug, test, and monitor your LLM aps built with LangGraph — read more about how to get started in the <a href=\"https://docs.smith.langchain.com\">docs</a>. \n",
+    "     </p>\n",
+    " </div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## How to build the graph\n",
```

Comment:
consider changing to just "Build the graph" -- i find it a bit confusing to see "how to" in subheaders + breaks symmetry w/ other headers